templates/components/login_socials.html.twig

```diff
@@ -16,4 +16,8 @@
         <a href="{{ path('oauth_keycloak_connect') }}" class="btn btn__secondary"><i class="fa-solid fa-lock" aria-hidden="true"></i>
             Keycloak</a>
     {% endif %}
+    {% if this.zitadelEnabled %}
+        <a href="{{ path('oauth_zitadel_connect') }}" class="btn btn__secondary"><i class="fa-solid fa-lock" aria-hidden="true"></i>
+            Zitadel</a>
```

Comment:
Could we get the icon of the zitadel org here? If it is too much effort that is fine. Maybe the Zitadel Logo itself (if they have a monochrome svg) would suffice?

Reply:
I don't think that's plausible. I checked my own instance and the url is `https://auth.incremental.social/ui/login/resources/dynamic?orgId=231964622825979915&default-policy=true&filename=policy/label/logo-dark-251434450057953312`. I have no idea how I'd get that org id though. That said, I'd urge against using the zitadel logo itself since typically instances are going to be using custom branding. In fact, I'm thinking perhaps the best thing to do would be adding 2 more env vars for the display name and logo to use for zitadel auth

Reply:
Of course it's possible. But currently we seem to use font-awesome icons (`fa`).  So now it's showing the `fa-lock` icon. 

However, font awesome doesn't have an icon / logo for Zitadel it seems. The icon seems to be come kind of triangle. so maybe `fa-caret-left` would do it for now?

![image](https://github.com/MbinOrg/mbin/assets/628926/0d436e69-e402-41cb-85c4-b4fca93d96ca)

Reply:
> That said, I'd urge against using the zitadel logo itself since typically instances are going to be using custom branding. In fact, I'm thinking perhaps the best thing to do would be adding 2 more env vars for the display name and logo to use for zitadel auth

I am fine with either just keeping the lock icon or the 2 env vars with logo and name. :shrug:

Reply:
But from the user perspective it makes sense that you know you are going to Zitadel. Right?

I don't care if Zitadel then support custom logins /logos or alike. But I digress

Reply:
I think for a first pass as is this is fine, thepaperpilot is the main user of it right now and likely already can write bespoke stuff if they haven't already. can always make it more customizable in a followup

Reply:
Sounds good to me